src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -7974,6 +7974,19 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="IDS_FeatureAllowsRefStructConstraint" xml:space="preserve">
     <value>allows ref struct constraint</value>
   </data>
+  <!-- PROTOTYPE: Condense -->
+  <data name="ERR_CannotApplyOverloadResolutionPriorityToOverride" xml:space="preserve">
+    <value>Cannot put 'OverloadResolutionPriorityAttribute' on an overriding member.</value>
```

Comment:
>Cannot put [](http://example.com/codeflow?start=11&length=10)

Perhaps "use" rather than "put", for more consistency with other attribute errors. For instance, "Cannot use 'OverloadResolutionPriorityAttribute' on ..." or "OverloadResolutionPriorityAttribute' cannot be used on ...".